man/docker-build.1.md

```diff
@@ -134,22 +134,20 @@ option can be set multiple times.
    about valid tag names.
 
 **-m**, **--memory**=*MEMORY*
-  Memory limit
+   Memory limit
 
-**--memory-swap**=*LIMIT*
-   A limit value equal to memory plus swap. Must be used with the  **-m**
-(**--memory**) flag. The swap `LIMIT` should always be larger than **-m**
-(**--memory**) value.
+**--memory-swap**=*number*[*S*]
+   Combined memory plus swap limit; *S* is an optional suffix which can be one of **b**, **k**, **m**, or **g**.
 
-   The format of `LIMIT` is `<number>[<unit>]`. Unit can be `b` (bytes),
-`k` (kilobytes), `m` (megabytes), or `g` (gigabytes). If you don't specify a
-unit, `b` is used. Set LIMIT to `-1` to enable unlimited swap.
+   This option can only be used together with **--memory**. The argument should always be larger than that of **--memory**. Default is double the value of **--memory**. Set to **-1** to enable unlimited swap.
 
-**--network**=*bridge*
+**--network**=*type*
   Set the networking mode for the RUN instructions during build. Supported standard
-  values are: `bridge`, `host`, `none` and `container:<name|id>`. Any other value
+  values are: **none**, **bridge**, **host** and **container:**_name_|_id_. Any other value
```

Comment:
I think the pointy-brackets are used as convention to indicate the value is _required_, so perhaps this should be 
```suggestion
  values are: **none**, **bridge**, **host** and **container:**<_name_|_id_>. Any other value
```

Alternatively you could split into two examples (`**container:**_name_ or `**container:**_id_`)